HPXMLtoOpenStudio/measure.rb

```diff
@@ -2770,6 +2772,24 @@ def self.set_foundation_and_walls_top()
     end
     @walls_top = @foundation_top + 8.0 * @ncfl_ag
   end
+
+  def self.set_heating_and_cooling_seasons(model)
+    return if @hpxml.hvac_controls.size == 0
+
+    hvac_control = @hpxml.hvac_controls[0]
+
+    htg_start_month = hvac_control.seasons_heating_begin_month
+    htg_start_day = hvac_control.seasons_heating_begin_day
+    htg_end_month = hvac_control.seasons_heating_end_month
+    htg_end_day = hvac_control.seasons_heating_end_day
+    clg_start_month = hvac_control.seasons_cooling_begin_month
+    clg_start_day = hvac_control.seasons_cooling_begin_day
+    clg_end_month = hvac_control.seasons_cooling_end_month
+    clg_end_day = hvac_control.seasons_cooling_end_day
+
+    @heating_season = HVAC.get_daily_season(model, htg_start_month, htg_start_day, htg_end_month, htg_end_day)
+    @cooling_season = HVAC.get_daily_season(model, clg_start_month, clg_start_day, clg_end_month, clg_end_day)
```

Comment:
Let's update these variables to be named more explicitly: "heating_days" and "cooling_days".